snapcraft/__init__.py

```diff
@@ -115,7 +115,11 @@
 import os
 import shutil
 
+from snapcraft._help import topic_help  # noqa
+from snapcraft._store import login, logout, upload  # noqa
 from snapcraft import common
+from snapcraft import lifecycle  # noqa
+from snapcraft import plugins  # noqa
```

Comment:
Because they are imported but not used, but by doing this you can just `import snapcraft` and get `snapcraft.plugins` and `snapcraft.lifecycle` already imported as well :smile:

Just like you get `os.path` when importing `os`